bedrock/firefox/templates/firefox/new/desktop/download.html

```diff
@@ -12,6 +12,12 @@
   {{ css_bundle('firefox_desktop_download') }}
 {% endblock %}
 
+{% block experiments %}
+  {% if switch('desktop-download', ['en-US']) %}
```

Comment:
We usually include "experiment" in switch names so they stand out in the list. Not required, just convention. So this could be `experiment-desktop-download` or even more verbose `experiment-firefox-desktop-download`